client_with_ssl.go

```diff
@@ -57,371 +56,136 @@ func main() {
 	defer client.Shutdown(ctx)
 	fmt.Println("Connection Successful!")
 
-	// Uncomment the example you want to run.
-	mapExample(client)
-	// sqlExample(client)
-	// jsonSerializationExample(client)
-	// nonStopMapExample(client)
-}
-
-func city(country string, name string, population int) serialization.JSON {
-	text := fmt.Sprintf(`{
-		"country":"%s", 
-		"city": "%s", 
-		"population":"%d"
-	}`, country, name, population)
-	return serialization.JSON(text)
-
-}
-
-func country(isoCode string, name string) serialization.JSON {
-	text := fmt.Sprintf(`{
-		"isoCode":"%s", 
-		"country": "%s"
-	}`, isoCode, name)
-	return serialization.JSON(text)
-}
-
-func mustPut(_ interface{}, err error) {
-	if err != nil {
-		panic(err)
-	}
+	createMapping(client.SQL())
```

Comment:
Is there a reason why this function takes the SQL service but other functions just the client itself?